.github/workflows/go.yml

```diff
@@ -94,3 +94,48 @@ jobs:
           fail_ci_if_error: true
           files: ./coverage.txt
           verbose: true
+
+  codeql:
+    name: CodeQL
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'go' ]
+        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python' ]
+        # Learn more:
+        # https://docs.github.com/en/free-pro-team@latest/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning#changing-the-languages-that-are-analyzed
```

Comment:
Do we really need all these CodeQL-related comments in every repo? If we miss something, then we can go to CodeQL action documentation and take a look there.

Reply:
That's the way it adds an action by default. Copy/pasting is much easier than thinking.